src/nccl_ofi_net.c

```diff
@@ -624,7 +621,7 @@ static int get_ofi_provider(char *prov_include, struct fi_info **prov_info_list)
 		}
 	}
 
-	return ret;
```

Comment:
We shouldn't return ncclSystemError if this was a successful completion.

Reply:
Bleh, I meant to return `ncclSuccess`, fixing.